content/en/tracing/serverless_functions/_index.md

```diff
@@ -0,0 +1,222 @@
+---
+title: Trace Serverless Functions
+kind: documentation
+description: 'Send traces from your Serverless Functions to Datadog'
+further_reading:
+    - link: 'serverless'
+      text: 'Serverless Monitoring with Datadog'
+---
+{{< img src="tracing/serverless_functions/ServerlessDistributedTrace.png" alt="Trace Serverless Functions"  style="width:100%;">}}
+
+In hybrid architectures, Serverless functions can form an important core of requests your application processes. Including these functions in your Distributed Traces can be critical for detecting performance bottlenecks and service outages in your distributed systems.
+
+## Install the AWS Integration
+
+As a first step, make sure to [install the AWS integration][1] before proceeding.
+
+Next, depending on your language and configuration, choose between using the Datadog Lambda Library or Amazon X-Ray for your traces.
+
+{{< img src="tracing/serverless_functions/LambdaLibraryXRay.png" alt="Lambda Library or X-Ray"  style="width:100%;">}}
+
+## Use the Datadog Lambda Library
+
+For [Java][2], [Node][3], [Python][4], [Ruby][5] and [Go][6], the [Datadog Lambda Library][7] is recommended and has several options available for installation.
```

Comment:
We don't support Datadog tracing for .NET, Go and Java right now. This might be unclear here, because some customers probably think Lambda Library == tracing.